app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id].to_i)
+    @post = @user.posts
+  end
```

Comment:
- [ ] Kindly make sure you correct this above typo error as your app raises the  below error when trying to visit the `app/views/posts/index.html.erb` page
![Screenshot from 2023-03-17 19-10-46](https://user-images.githubusercontent.com/83042742/225972846-1ea74736-234c-44a0-8adb-68bcf50d4ead.png)

```ruby
  # Good
    ...
    @posts = @user.posts
  end